### PR TITLE
build(publish): optimise npm package size

### PR DESCRIPTION
**Pull request by ilyamore88**
I've changed the `.npmignore` file because we have redundant files in our package.

image

https://www.npmjs.com/package/@editorjs/editorjs?activeTab=explore

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -1,22 +1,6 @@
...
-tsconfig.json
-tslint.json
-webpack.config.js
-yarn.lock
-devserver.js
+*
+!dist/**/*
+!types/**/*
+!LICENSE
+!README.md
```
Comment (neSpecc) [inline .npmignore:5]: readme contains some pictures from the `assets` as well

Comment (ilyamore88) [reply .npmignore:5]: Do we need it in the package? We need to use S3 for images then

**Review by neSpecc — APPROVED** [review]

**Review by gohabereg — APPROVED** [review]

**Comment by neSpecc** [comment]
Add a change log please

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -5,8 +5,9 @@
 - `Refactoring` — Popover class refactored.
 - `Improvement` — *Toolbox* — Number of `close()` method calls optimized.
 - `Improvement` — The `onChange` callback won't be triggered only if all mutations contain nodes with the `data-mutation-free` attributes.
-- `Fix` — Resolve compiler error from importing the BlockToolData type
-- `Fix` — Resolved a problem when document was being scrolled to the beginning after moving up a Block above the viewport
+- `Fix` — Resolve compiler error from importing the BlockToolData type.
+- `Fix` — Resolved a problem when document was being scrolled to the beginning after moving up a Block above the viewport.
+- `Build` — Remove redundant files from the package.
```
Comment (neSpecc) [inline docs/CHANGELOG.md]:
```suggestion
- `Improvement` — Package size reduced by removing redundant files
```

**Review by neSpecc — APPROVED** [review]

**Pull request merged**